internal/pkg/model/orchestrator.go

```diff
@@ -5,14 +5,14 @@ import (
 )
 
 type Orchestration struct {
-	Phases []Phase
+	Phases []*Phase
```

Comment:
Pre `Phase/Task` budeme pouzivat pointer.
Kedze bez toho sa s tym blbo pracuje, napr. pri premenovani `Phase` (dalsie PRs).

Podobne je to aj v transformation `Block/Code`:
https://github.com/keboola/keboola-as-code/blob/9a2ef792834b6baa520c699dcfafd00198432609/internal/pkg/model/object.go#L160-L173